release/jobs/health_monitor/templates/health_monitor.yml.erb

```diff
@@ -141,6 +141,12 @@ plugins:
       <% end %>
   <% end %>
 
+   <% if p("hm.syslog_event_forwarder_enabled") %>
```

Comment:
add default value here.

until we switch to bosh-init defaults are not respected by `bosh micro deploy`.